common/src/test/java/org/opentestsystem/rdw/admin/common/test/support/PermissionScopes.java

```diff
@@ -16,11 +16,11 @@ public static PermissionScope statewide() {
     }
 
     public static PermissionScope districts(final Long... ids) {
-        return new PermissionScope(Arrays.asList(ids), null);
+        return new PermissionScope(Arrays.asList(ids), null, null, null);
```

Comment:
uh oh i think we broke our self-imposed 3 param limit and need a builder now : p

Reply:
Dang, i was hoping nobody would catch that.

Reply:
Hard to miss with all those consecutive nulls. I thought you were instantiating an array x J

Reply:
I believe I said 5 to start and someone (.... @malaffoon ) thought 3 was better